aws/rust-runtime/aws-sigv4/Cargo.toml

```diff
@@ -33,6 +33,7 @@ percent-encoding = { version = "2.1", optional = true }
 regex = "1.5"
 ring = { version = "0.17.5", optional = true }
 sha2 = "0.10"
+crypto-bigint = "0.5"
```

Comment:
Should be optional, enabled by the sigv4a feature.

Can the `num-bigint` dependency be removed now?